### PR TITLE
FISH-5964: upgrading jakartaee-web-api for server

### PR DESCRIPTION
**Pull request by breakponchito**
Upgrading concurrency dependencies
## Description

this is to upgrade concurrency dependencies for 3.0 and be included on the web-profile
## Important Info
The 10.0.0-SNAPSHOT version is temporal and should be upgraded when the release of the jakarta 10 is done. For now this can be tested by building the last version of the jakarta api project here: https://github.com/eclipse-ee4j/jakartaee-api
### Testing Performed

Tested with a sample Web application with the jakarta.jakartaee-web-api dependency with provided scope into the payara-web distribution
### Testing Environment

Windows 11, Azul JDK 11

### CODE REVIEW

File: appserver/tests/payara-samples/samples/concurrency/pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
         <maven.compiler.target>8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <failOnMissingWebXml>false</failOnMissingWebXml>
-        <jakartaee-api.version>9.1.0</jakartaee-api.version>
+        <jakartaee-api.version>10.0.0-SNAPSHOT</jakartaee-api.version>
```
Comment (Pandrex247) [inline appserver/tests/payara-samples/samples/concurrency/pom.xml:60]: Is this property not redundant?  It's already defined at the top-level pom

File: appserver/tests/payara-samples/samples/concurrency/pom.xml [hunk 2]
```diff
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>jakarta.platform</groupId>
             <artifactId>jakarta.jakartaee-api</artifactId>
-            <version>10.0.0-RC1</version>
+            <version>${jakartaee-api.version}</version>
```
Comment (Pandrex247) [inline appserver/tests/payara-samples/samples/concurrency/pom.xml:97]: Is this required? It's already defined in dependency management in payara-bom

File: appserver/tests/payara-samples/samples/concurrency/pom.xml [hunk 3]
```diff
@@ -94,7 +94,7 @@
         <dependency>
             <groupId>jakarta.platform</groupId>
             <artifactId>jakarta.jakartaee-api</artifactId>
```
Comment (Pandrex247) [inline appserver/tests/payara-samples/samples/concurrency/pom.xml:96]: Should this not be using the web api now?

**Review by Pandrex247 — COMMENTED** [review]
Just a couple of comments on the dependencies

File: appserver/tests/payara-samples/samples/concurrency/pom.xml (hunk 1 above)
Comment (breakponchito) [reply appserver/tests/payara-samples/samples/concurrency/pom.xml]: yes it is redundant, now removed

File: appserver/tests/payara-samples/samples/concurrency/pom.xml (hunk 2 above)
Comment (breakponchito) [reply appserver/tests/payara-samples/samples/concurrency/pom.xml]: yes it is not needed. Now removed

File: appserver/tests/payara-samples/samples/concurrency/pom.xml (hunk 3 above)
Comment (breakponchito) [reply appserver/tests/payara-samples/samples/concurrency/pom.xml]: this was removed

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**